src/View/SSTemplateParser.peg

```diff
@@ -287,6 +287,8 @@ class SSTemplateParser extends Parser implements TemplateParser
         if (isset($sub['Call']['CallArguments']) && isset($sub['Call']['CallArguments']['php'])) {
             $arguments = $sub['Call']['CallArguments']['php'];
             $res['php'] .= "->$method('$property', [$arguments], true)";
+        } elseif ($property === 'Count') {
+            $res['php'] .= "->$property()";
```

Comment:
Allows us to handle `Count` in a special way in the `__call()` method of the data presenter layer.
This changes the template code from

```php
$val .= $scope->locally()->obj('Iterable', null, true)->XML_val('Count', null, true);
```

to 

```php
$val .= $scope->locally()->obj('Iterable', null, true)->Count();
```